src/Intro/Intro.jsx

```diff
@@ -47,12 +47,16 @@ const Intro = props => (
 	<div>
 		<section id="intro" className={props.classes.section}>
 			<Hidden smUp>
-				<ResponsiveImage path="/images/skystars.jpg" asBackgroundImage />
+				<ResponsiveImage
+					className={props.classes.fullVideo}
```

Comment:
I don't think `className={props.classes.fullVideo}` was needed. 
The prop `asBackgroundImage` will add the styles. 

I think it worked, maybe you found a bug or css modules stoped working

Reply:
The classname is needed for alignment on mobile, please check if you can reproduce. Without this class on mobile I got the image full-width but not cropped, so it would only fill op 1/3 of the screen.